### ISSUE TITLE
German translation confusing

### ISSUE THREAD

**Issue by Shellfishgene** [kingToolbox/WindTerm#277]
Hi! Thanks for a new release. It's nice that there are different languages, but in the case of German (and probably others) the machine translation is very confusing and not good enough. New languages should probably only be added when some native speaker has checked them.

**Comment by SlaviusPrime** [comment]
How switch to English?
Russian too bad ...

**Comment by Shellfishgene** [comment]
> How switch to English? Russian too bad ...

Session, Preferences, Settings, Appearance, then restart.

**Comment by kingToolbox** [comment]
@Shellfishgene, I totally agree with you. The reason why the local language is used by default now is to let users know the existence of the multilingual interface, because there are not many users who will read lengthy changelog carefully. If the user is not satisfied with the current machine translation, the user can also easily switch back to `English`.

The new translation files will be committed to the repository soon so that the community can assist in the translation, because I really can't complete the translation in dozens of languages by myself. :smile:

**Comment by KarelWintersky** [comment]
@kingToolbox where is language files?
